cloudferrylib/os/identity/keystone.py

```diff
@@ -51,17 +51,18 @@ class AddAdminUserToNonAdminTenant(object):
         your_operation_from_admin_user()
     """
 
-    def __init__(self, keystone, admin_user, tenant, member_role='admin'):
+    def __init__(self, keystone, admin_user, tenant_name, member_role='admin'):
```

Comment:
Method accepts both name and ID, which is stated in the docstring, please keep `tenant` as method argument